src/App.js

```diff
@@ -5,25 +5,27 @@ import UserOutput from './UserOuput/UserOutput';
 
 class App extends React.Component {
   state = {
```

Comment:
This will work, but usually React develpers use a constructor, it is allows to use values from props as default/initial values 
```
  constructor(props) {
    super(props)
  
    this.state = {
       username: props.username || 'Unknown'
    }
  }
```

Reply:
I tried to do the same without constructor using
```
state = {
    username: this.props.username || 'Unknown'
  }
```
 and it worked the same way if I understood it right.